pkg/kubelet/apis/config/types.go

```diff
@@ -407,6 +407,12 @@ type KubeletConfiguration struct {
 	// class value that lies in the range of that value and the next higher entry in the
 	// list when the node is shutting down.
 	ShutdownGracePeriodByPodPriority []ShutdownGracePeriodByPodPriority
+	// ShutdownSetPodsPhaseToTerminal sets pods to terminal phase during shutdown (enabled by default).
+	// If a node becomes ready again after graceful node shutdown (e.g. a reboot), then having ShutdownSetPodsPhaseToTerminal enabled, will result in pods not being started on the same node since the pods will be placed into terminal phase. In contrast, if ShutdownSetPodsPhaseToTerminal is disabled, pods will not be placed into terminal phase, and as result they will resume to run after the node becomes ready again.
```

Comment:
Please reformat this line to not exceed ~80 chars.

Reply:
Done.